src/content/guides/packages.md

```diff
@@ -178,6 +178,78 @@ due to conflicting version constraints in the pubspec.
 To identify out-of-date packages that require editing the pubspec,
 use [`dart pub outdated`][outdated].
 
+## Getting dependencies in production - `pub get --enforce-lockfile`
```

Comment:
Can we keep the header simpler so it's easier to parse in the TOC?

Also I'm assuming most developers aren't getting dependencies "in production" but rather "for production".

```suggestion
## Get dependencies for production
```